### ISSUE TITLE
Package request: `einops`

### ISSUE THREAD

**Issue by Richienb** [googlecolab/colabtools#4739]
**Is your feature request related to a problem? Please describe.**

It's frustrating to keep having to install [`einops`](https://github.com/arogozhnikov/einops), which I use so often. It's a popular module that lets you rearrange tensors by taking as input the starting shape and ending shape, then automatically applying the necessary transformations.

For example:

```py
from einops import rearrange

output_tensor = rearrange(input_tensor, 't b c -> b c t')
```

**Describe the solution you'd like**

Include the `einops` package by default.

**Comment by mayankmalik-colab** [comment]
I agree. `einops` is widely used package. We will try to install by default soon. Tracked internally - b/356851498 . Thanks
